sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -355,6 +360,27 @@ static int mt8195_get_bar_index(struct snd_sof_dev *sdev, u32 type)
 	return type;
 }
 
+static struct snd_soc_acpi_mach *mt8195_machine_select(struct snd_sof_dev *sdev)
+{
+	struct snd_sof_pdata *sof_pdata = sdev->pdata;
+	struct snd_soc_acpi_mach *mach;
+
+	mach = &sof_mt8195_mach;
+	if (!mach) {
```

Comment:
this is impossible - as long as `sof_mt8195_mach` is a "normal real" object

Reply:
@lyakh  Thanks. I will remove it.